packages/@aws-cdk/aws-eks/test/sdk-call-integ-test-docker-app/app/sdk-call.js

```diff
@@ -0,0 +1,11 @@
+const sdk = require('aws-sdk');
+sdk.config.update({region: 'us-east-2'});
+
+var s3 = new sdk.S3();
+s3.createBucket({ Bucket: 'amazingly-made-sdk-call-created-eks-bucket'}, function(err) {
```

Comment:
I see this value: `amazingly-made-sdk-call-created-eks-bucket` is being used a couple of times. Is it possible to have it as a constant in a file?

Reply:
This is a good callout, but this constant is used in two different places; once in the docker image that creates the bucket, and once in the lambda that verifies that the bucket was created. Any code sharing would require those two to share an environment and I don't think that's worthwhile here. Unless there's a way to do C-style preprocessing with typescript for both files, but I don't think there is.